From 936b9e08e5379f2bd607c7273510c9d3d4b17777 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 19 Dec 2019 23:48:15 +0200
Subject: [PATCH] [SW-0] preboot: gaudi: finally move to sram

While reverting the patches that moved preboot to Scratch Pad SRAM, the
new functionality (namely ppboot) was not adjusted correctly and thus
led to ppboot still copying preboot to Scratch Pad SRAM.

Side note: the version preparation code cannot run in Scratch Pad SRAM
because of the unaligned access nature of the strings and therefore it
produced an empty output.

Finally move preboot to SRAM and also fix the versioning issue.

Change-Id: I1f1b112949f267c8d263ef51b3414e2d1acac5ad
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/common.h b/gaudi/common.h
index 44edcbb..b9b1a05 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -5,7 +5,7 @@
  * SPDX-License-Identifier:     GPL-2.0+
  */
 
-#define PREBOOT_TEXT_BASE		0x7FFBFE4000ull
+#define PREBOOT_TEXT_BASE		0x7FF0004000ull
 
 #define mmDMA_W_PLL_BASE		0x487000
 #define mmIF_W_PLL_BASE			0x488000
